modules/s2i/bash/artifacts/usr/local/s2i/assemble

```diff
@@ -14,3 +14,52 @@ source "${JBOSS_CONTAINER_JAVA_S2I_MODULE}/maven-s2i-overrides"
 # invoke the build
 maven_s2i_build
 
+#TODO: Clean this up to call the scripts instead
+
+shopt -s globstar
+echo "Setting environment variables"
+jarfile="/deployments/*.jar"
+libdir="/deployments/lib"
+$JAVA_HOME/
```

Comment:
stray line? results in the following at runtime:
`/usr/local/s2i/assemble: line 23: /usr/lib/jvm/java-17/: Is a directory`